File: BoJ/python/level5(1차원 배열)/평균은 넘겠지.py
import sys

c = int(input())

for i in range(c):
    a = list(map(int, sys.stdin.readline().split()))
    sum = 0
    
    # avg = sum(a[1:]) / a[0]
    for j in a[1:]:
        sum += j
    avg = sum / a[0]

    cnt = 0
    for k in a[1:]:
        if k > avg:
            cnt += 1
    
    per = cnt/a[0]*100
    #print('%.3f'%per+'%')
    print('{:0.3f}%'.format(per))

File: BoJ/Java/단계별로 풀어보기/level5(1차원배열)/평균은_넘겠지.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

public class 평균은_넘겠지 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int c = Integer.parseInt(br.readLine());

        for(int i = 0; i < c; i++) {
            String[] str = br.readLine().split(" ");
            int[] ary = new int[str.length];
            for(int j = 0; j < str.length; j++) {
                ary[j] = Integer.parseInt(str[j]);
            }

            int sum = 0;
            for(int j = 1; j < ary.length; j++) {
                sum += ary[j];
            }
            float avg = sum / ary[0];

            int overAvg = 0;
            for(int j = 1; j < ary.length; j++) {
                if(ary[j] > avg) {
                    overAvg += 1;
                }
            }       
            double student = overAvg / (double)ary[0] * 100;
            bw.write(String.format("%.3f", student) +"%\n");
        }       
        bw.flush();
        bw.close(); 
    }
}
